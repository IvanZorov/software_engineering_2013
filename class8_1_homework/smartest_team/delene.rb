require 'csv'
a = []

CSV.foreach(ARGV[0]) do |row|
if ARGV[1].to_i < ARGV[2].to_i
if row[0].to_i >= ARGV[1].to_i
if row[0].to_i <= ARGV[2].to_i
if row[0].to_i%ARGV[3].to_i == 0
a << [row[0].to_i]
end
end
end
end
end

CSV.open("#{ARGV[0].chomp(".csv")}_result.csv", "w") do |csv|
a=a.sort_by {|a,b| a}
a.each do |e|
csv << e
end
end
