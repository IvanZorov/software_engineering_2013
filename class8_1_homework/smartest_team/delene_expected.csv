60
69
87
87
